telemetry/metrics.go

```diff
@@ -192,15 +192,15 @@ func (m *Metrics) gatherPrometheus() (GatherResponse, error) {
 	buf := &bytes.Buffer{}
 	defer buf.Reset()
 
-	e := expfmt.NewEncoder(buf, expfmt.FmtText)
+	e := expfmt.NewEncoder(buf, FormatText)
 
 	for _, mf := range metricsFamilies {
 		if err := e.Encode(mf); err != nil {
 			return GatherResponse{}, fmt.Errorf("failed to encode prometheus metrics: %w", err)
 		}
 	}
 
-	return GatherResponse{ContentType: string(expfmt.FmtText), Metrics: buf.Bytes()}, nil
+	return GatherResponse{ContentType: FormatText, Metrics: buf.Bytes()}, nil
```

Comment:
This is not really compatible:
```
FormatText       = "text"
```

But
```
expfmt.FmtText Format = `text/plain; version=` + TextVersion + `; charset=utf-8`
```

I'll create a content type variable so that we keep it the same

Reply:
thanks, was debating on doing this or not